### ISSUE TITLE
Error in installation

### ISSUE THREAD

**Issue by tarak77** [broadinstitute/wot#36]
Hi,
I am trying to install wot using python 3.6. For some reason after installing the dependencies when I try to 
`pip install wot`
I get an error
```
Collecting wot
  Could not find a version that satisfies the requirement wot (from versions: )
No matching distribution found for wot
```

Any help would be great!

**Comment by joshua-gould** [comment]
I'm not sure what the issue is. What OS are you using? Are you using miniconda (https://docs.conda.io/en/latest/miniconda.html).

**Comment by tarak77** [comment]
I am using my Mac,

```
Taraks-MacBook-Pro:~ tarakshisode$ pip install wot
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Collecting wot
  Could not find a version that satisfies the requirement wot (from versions: )
No matching distribution found for wot
```

**Comment by joshua-gould** [comment]
wot only works with python 3. I suggest you install python 3 from miniconda.

**Comment by geoffschieb** [comment]
Hi Tarak,

We have just released WOT 1.0 along with a tutorial:
https://broadinstitute.github.io/wot/tutorial/

Best,
Geoff
